kustomization.yml

```diff
@@ -1 +1 @@
-./recipes/kustomization-all.yml
```

Comment:
Was it intentional to override the base kustomization file in the project?

Reply:
Actually, no - I pushed it by mistake